classes/02class/exercises/c02-network08/gepifanio/README.md

```diff
@@ -0,0 +1,44 @@
+# c02-network08
+
+## Commands Execution Output
+
+- Answer the following questions
+  - What is the difference between inbound x outbound?
+    ```
+    
+    Inbound means incoming traffic to the network/resource
+    Outbound means outcoming traffic to the network/resource to outside 
+
+    ```
+    
+  - What is going to happen by default? Everything allowed or denied?
+    ```
+    
+    By default everything will be allowed as the rule 100 is allowing all the traffic coming from the VPC, and also the rule 101 is allowing everything from any source, and the `*` (star) means that is denying all connections inbound and outbound after the rule 101, which will still be allowed as the rule 101 allows everything, so the `*` (start) rule is not being activated in this case.
```

Comment:
@alanlima Fixed, hopefully now I understood how it works 😄

Reply:
I am glad to hear that 🚀